creator-node/src/components/healthCheck/healthCheckComponentService.js

```diff
@@ -9,8 +9,11 @@ const { MONITORS } = require('../../monitors/monitors')
  * the current git SHA, and service version info.
  * @param {*} ServiceRegistry
  * @param {*} logger
+ * @param {*} sequelize
+ * @param {string=} randomBytesToSign optional bytes string to be included in response object
+ *    and used in signature generation
  */
-const healthCheck = async ({ libs } = {}, logger, sequelize) => {
+const healthCheck = async ({ libs } = {}, logger, sequelize, randomBytesToSign) => {
```

Comment:
set randomBytesToSign=null by default. not sure if this will affect the healthCheckVerbose below but better to set the default

Reply:
fixed